.github/workflows/test-all-packages.yml

```diff
@@ -40,6 +40,17 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
 
+  dependency-graph:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install graphviz
+        run: sudo apt install -y graphviz
+
+      - name: Check for cycles
+        run: scripts/check-dependency-cycles.sh 4
```

Comment:
Do you know the 4 existing cycles off-hand? If so, jot them down in reply, please?

Reply:
```
❯ scripts/graph.sh
Cycles detected. These lines appear only in the original graph and not the acyclic variant:
	"@agoric/governance" -> "@agoric/vats";
	"@agoric/inter-protocol" -> agoric;
	"@agoric/vats" -> "@agoric/inter-protocol";
	"@agoric/zone" -> "@agoric/vat-data";
```